src/components/App/App.tsx

```diff
@@ -1,23 +1,34 @@
 import './App.css';
 import logo from '../../assets/FinalMatrimaniaLogo.png'
+import {Route, Switch, NavLink} from 'react-router-dom'
+// Components
+import VenderDashboard from '../VenderDashboard/VenderDashboard'
+import WeddingDetails from '../WeddingDetails/WeddingDetails'
+import WeddingPhotoList from '../WeddingPhotoList/WeddingPhotoList'
+import LandingPage from '../LandingPage/LandingPage'
+
+
 
 function App() {
   return (
     <div className="appWrap">
       <header className="headerWrap">
         <img src={logo} className="logo" alt="Matrimania Logo"/>
       </header>
-      <section>Filter By:
-        <select>
-          <option value="0">All</option>
-          <option value="1">Upcoming</option>
-          <option value="2">Past</option>
-          <option value="3">Today</option>
-        </select>
-      </section>
-      <section className="weddingCardWrap">
-      </section>
-      <button className="addWeddingButton">+</button>
+      <Switch>
+        <Route path='/vender-dashboard'>
+          <VenderDashboard />
+        </Route>
+				<Route path='/:weddingId/photo-list'>
+          <WeddingPhotoList />
+        </Route>
+        <Route path='/:weddingId'>
+          <WeddingDetails />
+        </Route>
+        <Route exact path='/'>
+          <LandingPage />
+        </Route>
+      </Switch>
```

Comment:
Glad we got the route situation figured out.

Reply:
Yes - figuring out that we need to install both react-router-dom and types/react-router-dom was a journey